thepracticaldev/dev.to
# Add active_status_updated_at to the Broadcasts Table

## What type of PR is this? (check all applicable)


## Description
This PR adds an `active_status_updated_at` column to the Broadcasts table. In addition to this, it adds a place to display the time at which a particular Broadcast was last set to `active` in `/internal/broadcasts`, an `active_status_updated_a` attribute to the Broadcasts factory, and a model spec that tests these changes.

## Related Tickets & Documents
Closes #8276 

## QA Instructions, Screenshots, Recordings
### QA Instructions:
1. Navigate to `/internal/broadcasts` and ensure that there are Broadcasts, if not, create a new one (Broadcast `type` does not matter in this case).
![Screen Shot 2020-06-12 at 1 53 52 PM](https://user-images.githubusercontent.com/32834804/84541318-66fa6c80-acb4-11ea-89f5-d5418bfdeb56.png)
2. Set the Broadcast to `Active: true` if it is not already. Observe that the Broadcast's "Last Active On:" timestamp updates.
![Screen Shot 2020-06-12 at 1 54 18 PM](https://user-images.githubusercontent.com/32834804/84541333-6f52a780-acb4-11ea-8ffa-7db050665774.png)
3. Set the Broadcast to `Active: false` so that it is no longer active and observe that the "Last Active On:" timestamp displays the time for when you first set the Broadcast to `Active: true`.
![Screen Shot 2020-06-12 at 1 54 33 PM](https://user-images.githubusercontent.com/32834804/84541354-7d082d00-acb4-11ea-93fa-a6971b661e53.png)
4. Set the Broadcast to `Active: true` so that it is active again and observe that the "Last Active On:" timestamp displays the timestamp that correlates to the time that you set it to `Active: true` again (the current time).
![Screen Shot 2020-06-12 at 1 55 08 PM](https://user-images.githubusercontent.com/32834804/84541375-872a2b80-acb4-11ea-8d3d-0771a430e9ac.png)
5. Try to break things! 

## Added tests?


## Added to documentation?


## [optional] Are there any post-deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![Clocks](https://media.giphy.com/media/p0FeUCcB2IrPq/giphy.gif)
